src/middleware/index.js

```diff
@@ -34,17 +34,9 @@ middleware.regexes = {
 	timestampedUpload: /^\d+-.+$/,
 };
 
-const csurfMiddleware = csrf({
-	cookie: nconf.get('url_parsed').protocol === 'https:' ? {
-		secure: true,
-		sameSite: 'Strict',
-		httpOnly: true,
-	} : true,
-});
-
 middleware.applyCSRF = function (req, res, next) {
-	if (req.uid >= 0) {
```

Comment:
@barisusakli Looks like this was initially added to exclude spiders? Changing this so guests don't get a CSRF token either.

Reply:
Hmm thought guests need csrf token for posting/login etc.?

Reply:
I would've thought... but it seems to work :thinking:

Reply:
Reverted :grimacing: